native_client/javascript/Makefile

```diff
@@ -29,21 +32,28 @@ package.json: package.json.in
 		-e 's/$$(PROJECT_VERSION)/$(PROJECT_VERSION)/' \
 		package.json.in > package.json && cat package.json
 
-configure: deepspeech_wrap.cxx package.json
+npm-dev: package.json
+	# node-gyp@5.x behaves erratically with VS2015 and MSBuild.exe detection
+ifeq ($(TARGET),host-win)
+	npm install node-gyp@4.x
+endif
+	npm --no-color install --prefix=$(NPM_ROOT)/../ --ignore-scripts --force --verbose --production=false .
+
+configure: deepspeech_wrap.cxx package.json npm-dev
 	$(NODE_BUILD_TOOL) configure $(NODE_BUILD_VERBOSE)
 
 build: configure deepspeech_wrap.cxx
-	AS=$(AS) CC=$(CC) CXX=$(CXX) LD=$(LD) CFLAGS="$(CFLAGS)" CXXFLAGS="$(CXXFLAGS)" LDFLAGS="$(RPATH_NODEJS) $(LDFLAGS)" LIBS=$(LIBS) $(NODE_BUILD_TOOL) $(NODE_PLATFORM_TARGET) $(NODE_RUNTIME) $(NODE_ABI_TARGET) $(NODE_DIST_URL) rebuild $(NODE_BUILD_VERBOSE)
+	AS=$(AS) CC=$(CC) CXX=$(CXX) LD=$(LD) CFLAGS="$(CFLAGS)" CXXFLAGS="$(CXXFLAGS)" LDFLAGS="$(RPATH_NODEJS) $(LDFLAGS)" LIBS=$(LIBS) $(NODE_BUILD_TOOL) $(NODE_PLATFORM_TARGET) $(NODE_RUNTIME) $(NODE_ABI_TARGET) $(NODE_DIST_URL) --no-color rebuild $(NODE_BUILD_VERBOSE)
 
 copy-deps: build
 	$(call copy_missing_libs,lib/binding/*/*/*/deepspeech.node,lib/binding/*/*/)
 
 node-wrapper: copy-deps build
-	$(NODE_BUILD_TOOL) $(NODE_PLATFORM_TARGET) $(NODE_RUNTIME) $(NODE_ABI_TARGET) $(NODE_DIST_URL) package $(NODE_BUILD_VERBOSE)
+	$(NODE_BUILD_TOOL) $(NODE_PLATFORM_TARGET) $(NODE_RUNTIME) $(NODE_ABI_TARGET) $(NODE_DIST_URL) --no-color package $(NODE_BUILD_VERBOSE)
```

Comment:
On my machine I need `NODE_BUILD_TOOL` to be `npx node-pre-gyp` rather than just a bare `node-pre-gyp` (since it's installed locally).

Reply:
> On my machine I need `NODE_BUILD_TOOL` to be `npx node-pre-gyp` rather than just a bare `node-pre-gyp` (since it's installed locally).

I've always worked with `PATH` being setup correctly. I don't really like `npx`, because: I did not knew about that magic tool, and it does too much magic (might install stuff in your back).

Reply:
There's nothing here about `PATH`: https://github.com/mozilla/DeepSpeech/blob/master/native_client/README.rst#install-nodejs--electronjs-bindings

Should something be added?